development/creds.example.env

```diff
@@ -1,20 +1,30 @@
-NAUTOBOT_DB_PASSWORD=notverysecurepwd
-NAUTOBOT_REDIS_PASSWORD=notverysecurepwd
-NAUTOBOT_SECRET_KEY=r8OwDznj!!dci#P9ghmRfdu1Ysxm0AiPeDCQhKE+N_rClfWNj
+################################################################################
+# CREDS File: Store private information. Copied to creds.env and always ignored
+################################################################################
+# Nautobot Configuration Secret Items
 NAUTOBOT_CREATE_SUPERUSER=true
-NAUTOBOT_SUPERUSER_API_TOKEN=0123456789abcdef0123456789abcdef01234567
+NAUTOBOT_DB_PASSWORD=changeme
+NAUTOBOT_NAPALM_USERNAME=''
+NAUTOBOT_NAPALM_PASSWORD=''
+NAUTOBOT_REDIS_PASSWORD=changeme
+NAUTOBOT_SECRET_KEY='changeme'
+NAUTOBOT_SUPERUSER_NAME=admin
+NAUTOBOT_SUPERUSER_EMAIL=admin@example.com
 NAUTOBOT_SUPERUSER_PASSWORD=admin
+NAUTOBOT_SUPERUSER_API_TOKEN=0123456789abcdef0123456789abcdef01234567
 
-# Needed for Postgres, must match the values for Nautobot above
-PGPASSWORD=notverysecurepwd
-POSTGRES_PASSWORD=notverysecurepwd
+# Postgres
+POSTGRES_PASSWORD=${NAUTOBOT_DB_PASSWORD}
+PGPASSWORD=${NAUTOBOT_DB_PASSWORD}
 
-# Needed for Redis, must match the values for Nautobot above
-REDIS_PASSWORD=notverysecurepwd
+# MySQL Credentials
+MYSQL_ROOT_PASSWORD=${NAUTOBOT_DB_PASSWORD}
+MYSQL_PASSWORD=${NAUTOBOT_DB_PASSWORD}
```

Comment:
Should we make a distinction between Postgres and MySQL credentials?  It is rare that they use the same environmental variable as passwords right?

Reply:
Personally I would prefer to remove binding from Docker Compose configs to these `creds.env` and `development.env`, and rather specify environment variables for each compose service independently. (Remove key `env_file` and use `environment` instead). The current solution exposes all environment files to most services, even when they do not consume it. However such change is not necessary now IMHO.

I use proposed solution here:

https://github.com/nautobot/cookiecutter-nautobot-app-drift-manager/blob/develop/development/compose.yaml

Environment is loaded and altered when using `invoke`.

https://github.com/nautobot/cookiecutter-nautobot-app-drift-manager/blob/develop/tasks.py#L38
https://github.com/nautobot/cookiecutter-nautobot-app-drift-manager/blob/develop/development/default.env#L1